### DISCUSSION TITLE
Welcome to Shimmer-Governance Discussions!

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Shimmer-Governance#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iota12345** [comment]
The current proposal will make the Shimmer price rapidly increase and remain high precisely because of the incentive structure. Whether your politics likes it or not, it has to be attractive for people with a lot of money. It will bring in new funding, moon boys (yes!), interest from exchanges, more funding for new projects and all the code developed will be used on the IOTA mainnet increasing the IOTA token value too.

**Reply by rkollar** [reply]
I don't think we should base any decision on increased price, "moon boys" or what exchanges like. If that's your criteria then we may aswell just use a dog logo for the project.

**Reply by iota12345** [reply]
You've missed the point sadly. Forget I mentioned the words 'moon boy'. I'll edit them out if you like?!

The point is that the Shimmer token needs to have value and become more valuable if the project is to work. So it needs to attract wealthy investors and reward them.

**Reply by 8bn-io** [reply]
The proposal will attract a number of entities with deep funding, but the current proposal seems to attract "moon boys" and split the established "IOTA community". I see a risk that a permanent staking boost by Genesis NFTs from the start will create a severe imbalance. Additionally, I don´t see how the current proposal severes the community despite increasing prices for $SMR. There might be better option to achieve a "sustainable" goal.

**Comment by iota12345** [comment]
Shimmer is a test network. The real value will remain in IOTA which will be adopted by companies using the lessons from the test network as Dom and Hans said repeatedly yesterday.

If I pay 3k (for a t-shirt) to invest in a test network and the successful ideas move to IOTA, I would expect a serious reward on offer for that risk because it may not work; 8% or 15% of nothing is still nothing. It may seem unfair (doesn’t to me) but it will fund development of the IOTA network. Down the track, we will all still have our shimmer tokens from the initial stake, which are fairly divided in proportion to one's initial IOTA investment. So if Shimmer does fly, we all still make money.

Regarding the time limit suggestion, would you invest in something that might go down in value and was guaranteed to stop paying you dividends as well? Should seed fund investors only expect company dividends for a couple of years? Or should it be every year they keep their shares? The point is- if it is made really unattractive at the seed funding stage, no-one will buy the tokens and then we will have no funding behind Shimmer and a token with no value. 

The original proposal is genius and delivers exactly what we currently require. You’re never going to please everyone and it’s time to simply lead and push this through. We need to behave more like other successful cryptos.

This is a critical point for IOTA: Not every decision needs to be put to a community vote. Dom must continue to lead from the front and not backtrack based on the views of some vocal community members on discord (out of hundreds of thousands of IOTA hodlers worldwide). It will look very weak if Dom does a U-turn and probably kill future funding of the Shimmer network, as investors will think that the leadership is rudderless, turn hype in FUD and make us the laughing stock of the crypto sphere.

**Reply by RetlawO** [reply]
It now has been multiple times stated by IF members, that it is also likely Shimmer will outperform IOTA on the longterm. There is no commitment by the IF to declare IOTA as the number one priority, not on protocol nor parthership level.

**Reply by iota12345** [reply]
It's also been said by IF members that it may fail completely and need to be forked.

**Comment by RetlawO** [comment]
Ask questions you’re wondering about:

1) Why is the NFT decided without community input?
2) Why do we have to create a third pot of money? (Besides of EDF & Treasury)
As it looks, EDF is for reasearch only and Treasury needs to have ISCP on mainnet. That leads to the question 3) Why did the IF rush to launch Shimmer? Why not deploy ISCP on Mainnet first? It somehow feels like IF is compensating something and this causes the need of a third cash pot. 

Curious on the answers, thanks!

**Reply by iota12345** [reply]
1. Large organisations don't ask every employee in their organisation their opinion about future strategy. We rely on people on the board to make decisions and be accountable for them. I personally do not want everyone in the community to contribute equally to decisions because some opinions are from people who are ill-informed.

2. The IF did not rush to launch Shimmer. They have been working on it for the last year. I'd recommend listening to the youtube AMA yesterday where Dom explained why we need Shimmer. Hans said we can't deploy ISCP on mainnet yet until the tech has been thoroughly tested using Shimmer. Shimmer has to have some monetary value to generate development and interest. You can't use IOTA treasury (unclaimed tokens) to provide funding for Shimmer because the IF would have to sell IOTA that they don't own or use IOTA as security (when they don't own the tokens) which isn't possible. So a different fund is required. 

3. I honestly wish people would just get behind them rather than dragging us back every time with all these silly unfounded suspicions.  There's no conspiracy! I can totally understand Dom's frustration and I think he should proceed with the original plan. If people don't like it because it offends their politics/ sense of entitlement/ opinion, they can sell their IOTA and watch from the sidelines.

**Reply by rkollar** [reply]
@iota12345  The entire point of a cryptocurency is to eventually get decentralized (both at the network level and the governance level). It is absolutely critical for people to voice their opinion and have the ability to make choices together. 

If the entire thing will stay controlled by a single organization then the project is literally pointless.

**Reply by iota12345** [reply]
That is my hope for IOTA too but we are not at that point yet are we. We are still working towards it. So we still need strong leadership. We don't need a democratic vote about every aspect of the journey. This is slowly turning into a fiasco where every man and his dog has an opinion about how to skin a cat and wants a vote on their method. We can't make efficient decisions like that.

**Reply by RetlawO** [reply]
@iota12345 not sure if this is the right project for you then. IOTA aims for the future usage of DAOs. 

1. "because some opinions are from people who are ill-informed"  -> its not about "some opinions" its about swarm intelligence, and your personal opinion is wrong here again (baseline is IOTA wants to use DAOs)

To the other stuff: you are ways too biased to have a neutral discussion, also I am quite sure now you fail to understand the whole bigger picture. 

Let's see if other people are more open for discussion regarding my questions here, else time will tell.

**Reply by iota12345** [reply]
A laughable response. Arrogant and biased. I'm only talking about the present Shimmer issue (now, not all future governance!). You've missed the point. Ironically you’ve completely exemplified my point about why the IF shouldn’t pander to the ramblings of a few vocal ill-informed outliers. Good luck.

**Comment by IGit666** [comment]
I'm wondering about the mechanics of the lottery distribution for the 1,111 airdropped nfts and the caps of number of nfts for any individual wallet. The minimum number of iota to qualify for an airdrop seems to incentivize splitting a wallet into multiple smaller wallets with at least 1gi to increase chances to get an nft. Splitting up a wallet in this way also seems to get around the 3 nft cap (this depends on how the purchasing will work, how much identifying information has to be given, ip addresses, etc. to prevent any one person from going around the cap). 

Although the digital identity capability is not up and running yet, this is the perfect use case. You could limit airdrops to DIDs, then whether you had one giant wallet with 1000Gi or 1000 wallets with 1Gi, you'd have the same chance and the same limits would apply. It would reassure smaller holders that whales can't get disproportionate share of airdropped nfts. 

My understanding of DID may have some gaps, but I'm wondering what other's thoughts are on this aspect of the nft launch. I'm not as concerned by the staking bonuses here (because they are already being thoroughly discussed, they definitely have high importance) as I am the part about access to future nfts since it seemed the most nebulous.
